lib/spack/spack/solver/concretize.lp

```diff
@@ -55,6 +55,7 @@ unify(SetID, PackageName) :- unification_set(SetID, node(_, PackageName)).
 unification_set("root", PackageNode) :- attr("root", PackageNode).
 unification_set(SetID, ChildNode) :- attr("depends_on", ParentNode, ChildNode, Type), Type != "build", unification_set(SetID, ParentNode).
 
+% What does this mean?
```

Comment:
This means that a "build" dependency moves across unification sets (from `SetID` of the parent node to `("build", node(X, Child))` of the child node).
```suggestion
```